gas/testsuite/gas/riscv/rvp_csr.d

```diff
@@ -0,0 +1,13 @@
+#as: -march=rv32gc_zpn -mabi=ilp32
```

Comment:
This part was before the vector extension, currently no needed about it, thanks :)